pages/demo.js

```diff
@@ -293,25 +296,24 @@ export default function Demo() {
                   </div>
                 )}
                 {!balanceChanging && (
+                  // eslint-disable-next-line
```

Comment:
What rule is this disabling out of interest?

Reply:
`jsx-a11y/no-static-element-interactions` and `jsx-a11y/click-events-have-key-events`